javascript/preflight.js

```diff
@@ -23,7 +23,11 @@ function runPreFlightChecks() {
         ghUrl             = 'https://api.github.com/rate_limit',
         authGhUrl         = ghUrl + '?access_token=' + token;
```

Comment:
we should remove the access token here.

Reply:
Good catch - I noticed this at some point but completely forgot to change it.  I'll force push a fix to keep the commit history clean.